include/industry_standard/spdm.h

```diff
@@ -439,7 +439,9 @@ typedef struct {
     uint16_t length;
 } spdm_get_certificate_request_t;
 
-#define SPDM_GET_CERTIFICATE_REQUEST_SLOT_ID_MASK 0xF
+#define SPDM_GET_CERTIFICATE_REQUEST_SLOT_ID_MASK        0xF
+#define SPDM_SET_CERTIFICATE_REQUEST_SETCERTMODEL_MASK   0x70
+#define SPDM_SET_CERTIFICATE_REQUEST_ERASE_MASK          0x80
```

Comment:
Please use `SPDM_SET_CERTIFICATE_REQUEST_ERASE`

Reply:
Thanks. I have changed the code.